2023-02-17-pulp-intro.md

```diff
@@ -8,43 +8,51 @@ Plugin Copr needs: pulp_rpm
 Glossary
 ========
 
-- Distribution
+- Artifact
 
-    Represents a user consumable content, on a http://some/path/ address
+    A data blob identified by sha256
 
-    Points at the last *repository version* through the last *publication*
+- Content
 
-- Publication
+    points at Artifact, but contains additional metadata
 
-    Point in time when we decide we want to publish a repository changes
-    in *distribution*.
+- Repository
 
-    (*distribution* points at a concrete *publication*, and it determines
-    a concrete *repository version*).
+    Named abstraction for a "repository".  When we add/remove *content*,
+    we don't add to a specific repository version.  We add it to
+    a *repository* and new *version* is generated.
 
 - Repository version
 
-    Represents *repository* state in a concrete time.  When a new
-    *content* (rpm) is uploaded and added to *repository*, new *version*
-    is generated.
+    Represents *repository* state in a specific time.  When a new
+    *content* (rpm) is uploaded and added to *repository*, new *repo
+    version* is generated automatically.
 
     Versions have a linear history, and are defined by set of *content*
     objects.
 
-- Repository
+    Creating a new *repository version* is relatively cheap; new RPM file
```

Comment:
This surprised me while trying to migrate a project into PULP.  Generating new repo version is cheap, but publication is relatively expensive.